feat(GuildIterator): fill `approximate_member_count` and `approximate_presence_count` fields when fetching Guilds (#892)
DisnakeDev/disnake: changelog/892.feature.rst

> Added ``with_counts`` field to :meth:`Client.fetch_guild` and :meth:`Client.fetch_guilds`, used to determine whether or not to include the guild's member count and presence information.

Reviewer: ```suggestion
Added ``with_counts`` parameter to :meth:`Client.fetch_guild` and :meth:`Client.fetch_guilds`, used to determine whether to include the guild's member count and presence information.
```
